package.json

```diff
@@ -33,6 +33,7 @@
     "eslint-plugin-react": "^7.23.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "fs-extra": "^10.0.0",
+    "nodemon": "^2.0.15",
```

Comment:
It seems like a good idea to delete debounces that aren't in use.

Reply:
https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/commit/5903e5b6fa6963a2868cceb9428eeeb0108d8939